Cookbook/Technical-Documents/Current-guide.md

```diff
@@ -0,0 +1,29 @@
+# `Current` Guide - how to use it in Babylon iOS Project
+
+### Where is that? 
+- `Current` is the global instance of the `World` type which is located in the `BabylonDependencies.framework`. 
+Its purpose is to provide convenient access to the instances that should be shared between different parts of the app (cross-cutting concerns). For more information please read the [How Control the World](/Cookbook/Proposals/ControlTheWorld.md) document.
```

Comment:
I was re reading this and I think we could improve this by saying what goes into Current and what the client can expect from it. Right now there is a link to the proposal, but the proposal might be outdated and not up to our current standards. So if this is the document that explains how to use Current it should also provide a tiny window into its contents.

Reply:
1.
> what goes into Current

Isn't this part clear? 
> convenient access to the instances that should be shared between different parts of the app

2.
> what the client can expect from it
I don't understand it. 
Again:
> ...access to the instances that should be shared...

3.
> might be outdated and 
the part that is outdated has been pointed by @inamiy  in [this comment](https://github.com/Babylonpartners/ios-playbook/pull/86#pullrequestreview-243068580) . I'm thinking how to cover that.

> not up to our current standards

I don't understand it. 

4. 
> it should also provide a tiny window into its contents
IMO pointing where to find is already enough to the Engineer find it's contents. But considering the tiny window, there is one already:
```swift 
let locale = Locale(identifier: "haw_US")
Current = World(autoupdatingLocale: locale)
```
I'm assuming (by the sequence of the Cookbook) that who's reading it, has already read the Proposal already mentioned. So no need to repeat.